src/main/java/org/team1540/elmo/ejectors/EjectInner.java

```diff
@@ -0,0 +1,22 @@
+package org.team1540.elmo.ejectors;
```

Comment:
This applies to all of these files, but I'm just going to write it once. It might be clearer what the difference between "InnerEjector" and "EjectInner" is if you rename it to "EjectInnerCommand" or "InnerEjectCommand"

Reply:
done